dapps/evancore.vue.libs/src/components/button/button.vue

```diff
@@ -26,7 +26,8 @@
       { [ `btn-${ size }` ]: size !== 'normal'  },
     ]"
     :disabled="disabled"
-    :href="href"
+    :href="href ? href : null"
```

Comment:
This looks weird. Can't add an optional be specified in the button.ts? like @Prop({type: String}) href?: string or like :href=href || null
???

Reply:
is changed to hrefs default logic